ts-embed_assets/static/excludeme.txt

```diff
@@ -0,0 +1 @@
+This file should be excluded
```

Comment:
```suggestion
This file should be excluded from calls to `<url_base>/static/...`
```

When I first read this, I thought it meant "from the repo" and that it was committed by mistake.